core/src/data_sources/splitter.rs

```diff
@@ -183,11 +183,22 @@ impl TokenizedSection {
             None => "",
         };
 
+        println!("Tokenizing section: {}", path);
+
         let (prefix, mut content) = try_join!(
             TokenizedText::from(embedder, section.prefix.as_ref()),
             TokenizedText::from(embedder, section.content.as_ref())
         )?;
 
+        match content.as_ref() {
+            Some(c) => {
+                println!("Tokenized content: tokens={}", c.tokens.len());
```

Comment:
should we leave those? if yes, maybe use `util::done`?

Reply:
yeah we can remove or move to utils::info :+1:  wilco